### DISCUSSION TITLE
Demo Session and Discussion on Interactive Programming/Computing (vs. TDD)?

### DISCUSSION THREAD

**Discussion by caichinger** [swkWien/sessions#130] [Announcements]
@gregorriegler 

Du hast gemeint dich würde das Thema interessieren?

Wenn du magst, können wir dazu einen Termin machen. Details TBD.
Ich poste hier, falls sich noch jemand anschließen mag.

Ich würde mit einer Demo anfangen, damit du ein Gefühl für die Arbeitsweise und (m)einen Workflow bekommst und dann von dort weiter schauen.

Für die Demo wären vielleicht zwei verschiedene Use Cases interessant:
* Data Science (da könnte ich ein Beispielprojekt aus meinem Schulungskontext nehmen)
* Irgendein Kata

Was sind deine Gedanken dazu?

---

Kommentare:
* Interactive Programming ist für mich zum Teil verwandt mit [Literate Programming und das ist ein relativ altes Konzept](https://en.wikipedia.org/wiki/Literate_programming).
* Ein frühes Tool um explizit Interactive Programming/Computing zu unterstützen ist [IPython (Interactive PythON)](https://en.wikipedia.org/wiki/IPython).

**Comment by ReaSu** [comment]
Ich würd mich gern anschließen, das klingt interessant.

Gehen Jupyter Notebooks auch in diese Richtung?

**Reply by caichinger** [reply]
Ja. Notebooks sind eine der Technologien mit denen man da gut arbeiten kann.

**Reply by caichinger** [reply]
Fein, dann machen wir da was dazu.

**Comment by gregorriegler** [comment]
Bin interessiert! @caichinger 
Ich fänds gut wenn du als der 'Instruktor' nicht tippen würdest, sondern @ReaSu oder ich

**Reply by caichinger** [reply]
Fein, machen wir das.

Format gerne TBD.

**Comment by caichinger** [comment]
### Terminfindung

Melde mich dazu noch separat.

Würde schauen, ob wir einen Raum im EPU Forum bekommen. Das stelle ich mir gerade nett vor.

**Reply by ReaSu** [reply]
Klingt gut! 

An den Reservierungsstunden solls nicht scheitern - ich hab noch nie einen reserviert. Aber dein Kalender ist wahrscheinlich voller, @caichinger.

**Reply by caichinger** [reply]
Zur Info: Wir werden das erst im neuen Jahr hinbekommen. Melde mich dann wieder.
